docs/source/reference/routines.rst

```diff
@@ -22,12 +22,13 @@ These functions cover a subset of
    linalg
    logic
    math
+   misc
```

Comment:
While it's not relevant to the PR, but the capitalization of the titles could be fixed.

> Logic Functions
> Mathematical Functions
> Miscellaneous functions

Reply:
Yeah, I'll fix this in #4945.